definitions/ext-rfscanner/golden_metric.yml

```diff
@@ -0,0 +1,39 @@
+# Determine metrics for VALET
+
+
+latency:
+  title: Latency  (ms)
+  displayAsValue: true
+
+
+  query:
+    select: average(totalTime) AS 'Latency'
+    from: THD_RF_TRANSACTION
+    facet: macAddress
+
+throughput:
+    title: Volume
+    
+    query:
+      select: count(*) AS 'Volume'
+      from: THD_RF_CONNECTION_EVENT
+      facet: macAddress      
```

Comment:
This will facet by `entity.name` and `macAddress` is that expected? or faceting by the entity name is enough?

Reply:
Since entity.name and macAddress are the same should we remove the facet: macAddress from every query?